internal/turso/turso.go

```diff
@@ -79,6 +82,13 @@ func (t *Client) newRequest(method, urlPath string, body io.Reader) (*http.Reque
 
 func (t *Client) do(method, path string, body io.Reader) (*http.Response, error) {
 	req, err := t.newRequest(method, path, body)
+	if flags.Debug() {
```

Comment:
Please make sure to check the error before you do anything with the return value.